SU-SWS/ansible-playbooks
# ACSF updates

# READY FOR REVIEW

# Summary
- Fixes to ACSF API
- Fixes to `stanford_dept` enabling the profile
- Ignore inventories in `inventory`
- Enable `nobots`; Disable `googleanalytics` and `pingdom_rum`
- Check to see if the site has been created successfully

# Needed By (Date)
- Monday, October 9th?

# Criticality
- How critical is this PR on a 1-10 scale? 2

# Steps to Test

1. Check out this branch
2. Populate an inventory
3. Run `ansible-playbook -i inventory/inventoryname migration-playbook.yml`

# Affected Projects or Products
- Sites 2.0

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SITES-126
- There will be one for `acsf-cardinal`
